### PR TITLE
KNOX-123 - Test commit

### PR DESCRIPTION
**Pull request by MrtnBalazs**
This is a test, do not merge!!

## What changes were proposed in this pull request?

Added a comment with test purpuses

## How was this patch tested?

TODO Not tested yet

### CODE REVIEW

File: gateway-util-common/src/test/java/org/apache/knox/gateway/audit/AuditLayoutTest.java [hunk 1]
```diff
@@ -115,6 +115,7 @@ public void testFullyFilledAuditEvent() {
         RESOURCE_TYPE, RESOURCE_NAME, OUTCOME, MESSAGE, System.lineSeparator() );
     String auditOutput = layout.toSerializable( event );
     assertThat( auditOutput, is( expectedOutput ) );
+    //hello world
```
Comment (zeroflag) [inline gateway-util-common/src/test/java/org/apache/knox/gateway/audit/AuditLayoutTest.java:118]: Test: Please add a space after the /

**Review by zeroflag — CHANGES_REQUESTED** [review]

File: gateway-util-common/src/test/java/org/apache/knox/gateway/audit/AuditLayoutTest.java (hunk 1 above)
Comment (MrtnBalazs) [reply gateway-util-common/src/test/java/org/apache/knox/gateway/audit/AuditLayoutTest.java]: Fixed

**Comment by moresandeep** [comment]
Looks good :) 
Looks good

**Comment by smolnar82** [comment]
This was a test PR.

**Pull request closed without merging**